### ISSUE TITLE
Fail gracely on unsupported platforms

### ISSUE THREAD

**Issue by mobidev111** [chariotsolutions/phonegap-nfc#213]
On windows 8.1 (not windows phone) the plugin fails with the following message:
```
   adding proxy for NfcPlugin
   WARNING: proximity device is null
   Registering for NDEF
   TypeError: Unable to get property 'subscribeForMessage' of undefined or null reference
   adding NFC Listener failed: TypeError: Unable to get property 'subscribeForMessage' of undefined or null reference
   Error in Error callbackId: NfcPlugin816380662 : adding NFC Listener failed: TypeError: Unable to get property 'subscribeForMessage' of undefined or null reference
   Exception calling native with command :: NfcPlugin :: registerNdef ::exception=adding NFC Listener failed: TypeError: Unable to get property 'subscribeForMessage' of undefined or null reference
   Initialized the NfcPlugin
```

-> better fail by testing for supported platforms

**Comment by JohnMcLear** [comment]
8.1 is working for me, can you provide your source code so we can replicate please?

**Comment by don** [comment]
Fixed in 0.6.6
